demo.cfg

```diff
@@ -1,2 +1,3 @@
 [buildout]
 extends = buildout.cfg
+primary_indexer_queue_type = REDIS_LIST
```

Comment:
This is just for testing, it should work on demos but the default should be 'simple'